src/stylesheets/settings/_settings-components.scss

```diff
@@ -76,6 +76,11 @@ $theme-input-max-width: "mobile-lg" !default;
 $theme-input-select-border-width: 2px !default;
 $theme-input-select-size: 2.5 !default;
 $theme-input-state-border-width: 0.5 !default;
+$theme-input-tile-background-color-selected: 'accent-cool-lighter' !default;
```

Comment:
Should these be `primary` family by default?

Reply:
I had preferred the brightness of the `accent-cool` color, but it's not strong preference.

**Accent**
<img width="371" alt="Screen Shot 2020-10-28 at 1 15 47 PM" src="https://user-images.githubusercontent.com/1094951/97472143-e1542200-191f-11eb-88ea-819c2434837c.png">


**Primary**
<img width="367" alt="Screen Shot 2020-10-28 at 1 14 42 PM" src="https://user-images.githubusercontent.com/1094951/97471913-9cc88680-191f-11eb-8cdd-0ed7b22f86ac.png">

Reply:
@thisisdano what's your preference? design system tends to default towards `primary`, right?

Reply:
I think it _looks_ better with `accent-cool` but I think the better default is `primary`

Reply:
Primary it is!